dbms/src/Core/Settings.h

```diff
@@ -395,6 +395,7 @@ struct Settings : public SettingsCollection<Settings>
     M(SettingBool, enable_early_constant_folding, true, "Enable query optimization where we analyze function and subqueries results and rewrite query if there're constants there", 0) \
     \
     M(SettingBool, partial_revokes, false, "Makes it possible to revoke privileges partially.", 0) \
+    M(SettingBool, force_deduplicate_childrens, false, "If the 'root' table deduplicates blocks, there are no need to make deduplication for children. Use true to always deduplicate childrens.", 0) \
```

Comment:
childrens -> children

Reply:
Need explanation that children are denendent materialized views.

force_deduplicate_childrens -> `deduplicate_blocks_in_dependent_materialized_views`